source/_components/smartthings.fan.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: "SmartThings Fan"
+description: "Instructions on setting up Samsung SmartThings fans within Home Assistant."
+date: 2018-02-01 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: samsung_smartthings.png
+ha_category: Fan
+ha_release: ""
```

Comment:
`0.87`

Reply:
Will the parent PR still be included in the 0.87 release?

Reply:
That is the hope (and looks like it should be) :)

Reply:
I'd prefer to see the `light` prioritized as that will be more valuable to the users and that PR fixes/standardizes a few things this one is assuming.  Getting both in would be good too though. :)

Reply:
Updated it to 0.87, assuming it will go in.